src/pane.coffee

```diff
@@ -514,7 +514,8 @@ class Pane extends Model
 
     saveOptions = item.getSaveDialogOptions?() ? {}
     saveOptions.defaultPath ?= item.getPath()
-    newItemPath = atom.showSaveDialogSync(saveOptions)
+    showSaveDialogSync = if item.showSaveDialogSync else atom.showSaveDialogSync
```

Comment:
Does this even work? Wouldn't it be `item.showSaveDialogSync ? atom.showSaveDialogSync`. Also the `this` would be `window` for this call. Not ideal.